lib/repl.js

```diff
@@ -567,6 +570,14 @@ function REPLServer(prompt,
         if (e.stack) {
           if (e.name === 'SyntaxError') {
             // Remove stack trace.
+            const importErrorStr = 'Cannot use import statement outside a module';
+            if (StringPrototypeIncludes(e.message, importErrorStr)) {
+              e.message = 'Cannot use import statement inside the Node.js repl, ' +
+                'please use dynamic import instead';
+              const splitStack = StringPrototypeSplit(e.stack, '\n');
+              splitStack[4] = `SyntaxError: ${e.message}`;
```

Comment:
we could alternative find which item in the array starts with `SyntaxError` rather than by default using the 5th item

Reply:
That sounds better to me. There is in fact similar code to determine that line below. It'll only be triggered in case there's no uncaughtException listener though.

Reply:
Updated to use regex PTAL